resources/assets/js/i18n/en.json

```diff
@@ -105,6 +105,8 @@
         "viewPost": "View Post",
         "viewProfile": "View Profile",
         "moderationTools": "Moderation Tools",
+        "muteUser": "Mute User",
```

Comment:
You should be adding translations to the `resources/lang/en/profile.php` file so they can be translated to other langs.

The `resources/assets/js/i18n/*.json` files are auto generated by the `resources/lang/en/*.php` files and should never be edited directly.